libnmstate/schemas/operational-state.yaml

```diff
@@ -177,6 +178,31 @@ definitions:
           required:
             - id
             - base-iface
+  interface-vxlan:
```

Comment:
What about `group` for multicast support? Isn't it basic?

Reply:
Remote can be unicast or multicast, that's why I don't add it like 'group'

https://developer.gnome.org/NetworkManager/stable/settings-vxlan.html